Lib/uuid.py

```diff
@@ -404,8 +404,8 @@ def _arp_getnode():
     # This works on Linux, FreeBSD and NetBSD
     mac = _find_mac('arp', '-an', [os.fsencode('(%s)' % ip_addr)],
                     lambda i: i+2)
-    if mac:
-        return mac
+    # Return None instead of 0.
+    return mac if mac else None
```

Comment:
And if pep8-izy the code, I suggest to add explicit `return None` in other functions.

For uniformity I would prefer
```python
    if mac:
        return mac
    return None
```
in this function.

Reply:
I like the code @serhiy-storchaka proposed.

Reply:
@vstinner Yep, agreed.